README.md

```diff
@@ -52,6 +52,9 @@ Only global configuration options are documented here.
 <td><code>nuntio.engine.serviceAddress.[ipv4|ipv6]</code></td><td><code>true</code></td><td>Which type of service addresses will be registered in the service registry.</td>
 </tr>
 <tr>
+<td><code>nuntio.engine.forcedTags</code></td><td><code>{}</code></td><td>Force-add comma-separated tags on all services</td>
```

Comment:
I would prefer simply `nuntio.engine.tags` but that might be a personal preference

Reply:
Why? I think `tags` is a bit vague in what it means, `forcedTags` is a bit more explicit in that the tags are forced to be present on all services.